app/src/main/res/values/strings.xml

```diff
@@ -7,15 +7,49 @@
 
     <string name="Dips_video">https://www.youtube.com/watch?v=VYgVTinbx_A</string>
     <string name="Skullcrushers_video">https://www.youtube.com/watch?v=ir5PsbniVSc</string>
+    <string name="BicepCurl_video">https://www.youtube.com/watch?v=QZEqB6wUPxQ</string>
```

Comment:
Mentioned it before but should be: <string name="BicepCurl_video">QZEqB6wUPxQ</string> instead. Everything with _video should be like that form expect pointing to its own video tag.